drivers/include/periph/uart.h

```diff
@@ -111,6 +111,44 @@ enum {
     UART_NOMODE     = -4    /**< given mode is not applicable */
 };
 
+/**
+ * @brief   Definition of possible parity modes
+ */
+#ifndef HAVE_UART_PARITY_T
+typedef enum {
+   UART_PARITY_NONE,   /**< no parity */
+   UART_PARITY_EVEN,   /**< even parity */
+   UART_PARITY_ODD,    /**< odd parity */
+   UART_PARITY_MARK,   /**< mark parity */
+   UART_PARITY_SPACE   /**< space parity */
+} uart_parity_t;
+#endif
+
+/**
+ * @brief   Definition of possible data bits lengths in a UART frame
+ */
+#ifndef HAVE_UART_DATA_BITS_T
+typedef enum {
+    UART_DATA_BITS_8,   /**< 8 data bits */
+    UART_DATA_BITS_5,   /**< 5 data bits */
+    UART_DATA_BITS_6,   /**< 6 data bits */
+    UART_DATA_BITS_7,   /**< 7 data bits */
+    UART_DATA_BITS_9    /**< 9 data bits. For now, there is no interface
```

Comment:
remove this option. Once/if we ever want to enable our interface to send more then 8 bit, we can add this value to the enum then!

Reply:
ACK